blueflood-core/src/integration-test/java/com/rackspacecloud/blueflood/service/ScheduleContextIntegrationTest.java

```diff
@@ -1,76 +0,0 @@
-/*
```

Comment:
Oh, I see that the 2 tests in ScheduledContext are related to ZooKeeper only. There are no other tests for ScheduledContext. 😱 I am fine adding the real test for ScheduledContext in a different story, if needed.